lava-slave-install.md

```diff
@@ -0,0 +1,93 @@
+This is an howto install a lava-docker slave
+This howto is based on a Debian host.
+
+# Create a dedicated user
+The user will be lavadocker in our example. It must be in the docker group for doing docker operations.
+```
+useradd -G docker lavadocker
+```
+
+# Install all pre-requisites packages
+docker-compose need to be installed from PIP
+```
+apt-get install git python-pip
+```
+
+# Install docker-ce
+See https://docs.docker.com/install/linux/docker-ce/debian/ for more detailled informations.
+
+# install docker-compose for our user
+
+As lavadocker run:
+```
+pip install --user docker-compose
+```
+
+# Install a second network interface
+For our example it will be named enx0
+
```

Comment:
This needs more details.

Why is the 2nd interface needed?  (e.g. so boards can be on a dedicated network).   Also, having the boards on a separate network is a good idea, but is not required, so this could be a recommended, but optional step.